src/__test__/BoxButton.test.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable comma-dangle */
```

Comment:
- [ ] Please kindly ensure that you did not disable linters, using a linter as part of your development process can lead to more efficient and reliable code writing, as it helps catch errors early and encourages adherence to coding best practices. I'm sure you can address the issues that these linters are pointing at without disabling them. ℹ️ 

**Kindly implement this for all your test files because I noticed that you disabled linters for all the test files.**